src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.cs

```diff
@@ -91,12 +109,12 @@ public void newline_is_legal_at_the_beginning_of_a_request()
             var result = HttpRequestParser.Parse(
                 """
         
-        GET https://example.com
-        """);
+                GET https://example.com
+                """);
 
             result.SyntaxTree.RootNode
                   .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
-                  .MethodNode.ChildTokens.First().TextWithTrivia.Should().Be("\r\n");
+                  .MethodNode.ChildTokens.First().TextWithTrivia.Should().Contain("\r\n");
```

Comment:
Can we assert that there are 3 tokens with `token.Kind is TokenKind.Newline` instead?

Reply:
Looks like this is now fixed.